src/diagnostics.toit

```diff
@@ -0,0 +1,34 @@
+// Copyright (C) 2021 Toitware ApS. All rights reserved.
+// Use of this source code is governed by a MIT-style license that can be found
+// in the LICENSE file.
+
+/**
+Diagnostics information for a GNSS receiver.
+*/
+class GnssDiagnostics:
```

Comment:
Perhaps just diagnostics? It's already scoped to the package (`ublox_gnss.Diagnostics`).